.github/workflows/continuous-integration-karma-edit-story.yml

```diff
@@ -76,7 +76,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           ref: gh-pages
-  
+
```

Comment:
[nit]: Is this change intended?

Reply:
Not necessarily but seems to be introduced by the linter, so I'd rather leave it (someone will have to merge it at some point.)